nixos/tests/omnom.nix

```diff
@@ -26,17 +31,116 @@ in
             server.address = "0.0.0.0:${toString servicePort}";
           };
         };
+
+        environment.systemPackages =
+          let
+            # https://addons.mozilla.org/en-US/firefox/addon/omnom/
+            addon = pkgs.fetchurl {
+              url = "https://addons.mozilla.org/firefox/downloads/file/4516876/omnom-0.23.0.xpi";
+              hash = "sha256-ixgJM2aGctWN51Xu+OeBGyv+V+OekbIPTh/PvsJZLh8=";
+            };
+            seleniumScript =
+              pkgs.writers.writePython3Bin "selenium-script"
+                {
+                  libraries = with pkgs.python3Packages; [ selenium ];
+                  flakeIgnore = [ "E501" ];
+                }
+                # python
+                ''
+                  import sys
+                  from selenium.webdriver import Firefox, FirefoxService
+                  from selenium.webdriver.common.by import By
+                  from selenium.webdriver.firefox.firefox_profile import FirefoxProfile
+                  from selenium.webdriver.firefox.options import Options
+                  from selenium.webdriver.support import expected_conditions as EC
+                  from selenium.webdriver.support.ui import WebDriverWait
+
+                  uuid = "2b09138b-b2a5-4e14-b938-4e8ff510941d"
+
+                  service = FirefoxService(executable_path="${lib.getExe pkgs.geckodriver}")
+                  profile = FirefoxProfile()
+                  # manifest.json
+                  profile.set_preference(
+                      "extensions.webextensions.uuids",
+                      '{"{f0bca7ce-0cda-41dc-9ea8-126a50fed280}": "' + uuid + '"}',
```

Comment:
Can we perhaps clarify what this does? What's the part that comes before the uuid and what does this setting do?